cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/SessionManager.java

```diff
@@ -196,11 +216,15 @@ synchronized void expireOldSessions() {
     private final String _requestUrl;
     private final Map<String, String[]> _requestParameters;
     private final List<OperationFuture> _operationFuture;
+    private final long _requestTime;
```

Comment:
Is it `_requestTimeNs`? Can we update the name and corresponding getter, comments, etc?

Reply:
sessionInfo is a private static inner class of sessionManager and this variable is a final variable. I feel that the getter/setter/comment maybe unnecessary.